core/src/impl/Morpheus_Multiply_Impl.hpp

```diff
@@ -0,0 +1,67 @@
+/**
+ * Morpheus_Multiply_Impl.hpp
+ *
+ * EPCC, The University of Edinburgh
+ *
+ * (c) 2021 The University of Edinburgh
+ *
+ * Contributing Authors:
+ * Christodoulos Stylianou (c.stylianou@ed.ac.uk)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * 	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef MORPHEUS_ALGORITHMS_IMPL_MULTIPLY_IMPL_HPP
+#define MORPHEUS_ALGORITHMS_IMPL_MULTIPLY_IMPL_HPP
+
+#include <impl/Serial/Morpheus_Multiply_Impl.hpp>
+#include <impl/OpenMP/Morpheus_Multiply_Impl.hpp>
```

Comment:
Shall we find a way to add these in configure time based on what has been compiled?